src/analysis/static/TargetPool.ts

```diff
@@ -0,0 +1,37 @@
+import { TargetFile } from "./TargetFile";
+import { CFG } from "./graph/CFG";
+
+export abstract class TargetPool {
+  private _included: TargetFile[];
+  private _excluded: TargetFile[];
+
+  abstract getSource(targetPath: string): string;
+  abstract getTargetMap(targetPath: string): Map<string, any>;
+  abstract getFunctionMap(
+    targetPath: string,
+    targetName: string
+  ): Map<string, any>;
+
+  abstract getCFG(targetPath: string, targetName: string): CFG;
+  abstract getAST(targetPath: string): any;
+
+  set included(value: TargetFile[]) {
```

Comment:
Remove these getters and setters in favor of a load targets method

Reply:
Done